### PR TITLE
901074 How to set the font size of text annotations greater than values available in WPF PdfViewer

### PR DESCRIPTION
**Pull request by KrithikaGanesan**
How to set the font size of text annotations greater than values available in WPF PdfViewer

### CODE REVIEW

File: Annotations/CustomizeFontSizeOfTextAnnotation/CustomizeFontSizeSample/CustomizeFontSizeSample.csproj [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Syncfusion.Compression.Base, Version=27.2462.2.0, Culture=neutral, PublicKeyToken=3d67ed1f87d44c89, processorArchitecture=MSIL">
+      <HintPath>packages\Syncfusion.Compression.Base.27.2.2\lib\net462\Syncfusion.Compression.Base.dll</HintPath>
+    </Reference>
+    <Reference Include="Syncfusion.Licensing, Version=27.2462.2.0, Culture=neutral, PublicKeyToken=632609b4d040f6b4, processorArchitecture=MSIL">
+      <HintPath>packages\Syncfusion.Licensing.27.2.2\lib\net462\Syncfusion.Licensing.dll</HintPath>
+    </Reference>
+    <Reference Include="Syncfusion.Pdf.Base, Version=27.2462.2.0, Culture=neutral, PublicKeyToken=3d67ed1f87d44c89, processorArchitecture=MSIL">
```
Comment (RajaVigneshB) [inline Annotations/CustomizeFontSizeOfTextAnnotation/CustomizeFontSizeSample/CustomizeFontSizeSample.csproj:46]: References should be general

File: Annotations/CustomizeFontSizeOfTextAnnotation/CustomizeFontSizeSample/CustomizeFontSizeSample.csproj [hunk 2]
```diff
@@ -0,0 +1,126 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{A263C8D9-D9AD-416A-B127-F98C84642E6A}</ProjectGuid>
```
Comment (RajaVigneshB) [inline Annotations/CustomizeFontSizeOfTextAnnotation/CustomizeFontSizeSample/CustomizeFontSizeSample.csproj:7]: Check whether it exist in all other samples.

File: Annotations/CustomizeFontSizeOfTextAnnotation/CustomizeFontSizeSample/MainWindow.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
```
Comment (RajaVigneshB) [inline Annotations/CustomizeFontSizeOfTextAnnotation/CustomizeFontSizeSample/MainWindow.xaml.cs:15]: Remove unwanted namespace

**Review by RajaVigneshB — CHANGES_REQUESTED** [review]

File: Annotations/CustomizeFontSizeOfTextAnnotation/CustomizeFontSizeSample/MainWindow.xaml.cs (hunk 3 above)
Comment (KrithikaGanesan) [reply Annotations/CustomizeFontSizeOfTextAnnotation/CustomizeFontSizeSample/MainWindow.xaml.cs]: I have removed unwanted namespaces

File: Annotations/CustomizeFontSizeOfTextAnnotation/CustomizeFontSizeSample/CustomizeFontSizeSample.csproj (hunk 2 above)
Comment (KrithikaGanesan) [reply Annotations/CustomizeFontSizeOfTextAnnotation/CustomizeFontSizeSample/CustomizeFontSizeSample.csproj:7]: Yes. It exists in other samples.

**Review by RajaVigneshB — APPROVED** [review]

**Pull request merged**